2-mandatory/4-stocks.js

```diff
@@ -34,8 +34,20 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  let newArr = [];
+  for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+    let avgStockPrice = 0;
```

Comment:
It is better to rename this variable avgStockPrice as it stores the accumulated sum of 5 days stock prices.